src/sentry/static/sentry/app/index.js

```diff
@@ -19,24 +19,17 @@ import moment from 'moment';
 
 import 'app/utils/emotion-setup';
 
-import {CSRF_COOKIE_NAME} from 'app/constants';
 import Main from 'app/main';
 import * as api from 'app/api';
-import getCookie from 'app/utils/getCookie';
+import ajaxCSRFSetup from 'app/utils/ajaxCSRFSetup';
```

Comment:
one last nit, with acronyms it'd be easier to be consistent if we treated them as words (so `ajaxCsrfSetup`). Good example here is what if it was `getAJAXCSRFSetup`